copy/tutorials/import-from-code/howtos.md

```diff
@@ -0,0 +1,21 @@
+### How to import code
+
+Your code should be formatted as a [`createMachine()` factory function](https://xstate.js.org/docs/guides/machines.html) before import. The importer has basic validation in case your machine has basic errors, including reminding you if the `createMachine` definition is missing.
+
+#### How to import code from the code panel
+
+Caution: importing code from the code panel will overwrite the current machine in the editor.
+
+1. Open the **Code** panel from the right tool menu.
+2. Use the **Import** button to open the importer.
+3. Paste or type your machine code into the text area.
+4. Use the **Create machine** button to import your machine.
```

Comment:
In addition to this, you can also use the left side panel and either right click on a machine in the list or press the **...** button to open the options menu and choose "import code".
<img width="397" alt="CleanShot 2022-11-28 at 20 10 27@2x" src="https://user-images.githubusercontent.com/8332043/204349973-ac839ba1-f0ff-4c7a-a711-7793ec0b5a82.png">
<img width="573" alt="CleanShot 2022-11-28 at 20 10 32@2x" src="https://user-images.githubusercontent.com/8332043/204350008-d7ca7890-7132-4187-a01b-74b1a6752622.png">